spec/unit/hanami/action_spec.rb

```diff
@@ -136,6 +136,16 @@ def call(_)
   end
 
   describe "#parsed_request_body" do
+    before do
+      @original_stderr = $stderr
+      $stderr = File.open(File::NULL, "w")
+    end
+
```

Comment:
@jodosha what do you think about what @AlfonsoUceda mentioned in of moving this functionality to `Hanami::Utils::IO`?

Reply:
@GustavoCaso Please have a look at https://github.com/hanami/utils/blob/develop/spec/support/silence_deprecations.rb

I'd add it to `hanami-devtools`, WDYT?